sdk/options.go

```diff
@@ -42,6 +42,8 @@ type Options struct {
 
 	config []byte
 	block  bool
+	// When calling the sdk the user can specify an opa id so that repeat calls to the sdk don't have a unique opa id
+	ID string
```

Comment:
Nit, but should this not be moved up to where the other exported fields are declared?

Reply:
will move it